pnpm-lock.yaml

```diff
@@ -1245,8 +1252,8 @@ packages:
     dev: false
     optional: true
 
-  /@next/env@13.3.0:
-    resolution: {integrity: sha512-AjppRV4uG3No7L1plinoTQETH+j2F10TEnrMfzbTUYwze5sBUPveeeBAPZPm8OkJZ1epq9OyYKhZrvbD6/9HCQ==}
+  /@next/env@13.3.4:
```

Comment:
@alexanderatallah All of these things updated from 13.3.0 to 13.3.4 when I installed tiktoken. Not sure if that was supposed to happen or not.